src/search/merge_and_shrink/merge_and_shrink_heuristic.cc

```diff
@@ -265,7 +264,8 @@ class MergeAndShrinkHeuristicFeature : public plugins::TypedFeature<Evaluator, M
             options_copy.get<int>("threshold_before_merge"),
             options_copy.get<bool>("prune_unreachable_states"),
             options_copy.get<bool>("prune_irrelevant_states"),
-            options_copy.get<double>("main_loop_max_time")
+            options_copy.get<double>("main_loop_max_time"),
```

Comment:
here we should also replace some .get calls with `get_merge_and_shrink_algorithm_arguments_from_options`